src/pages/questionPage.js

```diff
@@ -19,8 +19,29 @@ export const initQuestionPage = () => {
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
+  const selectAnswer = (answerElement) => {
+    const answers = document.querySelectorAll('li');
+    answers.forEach((answer) => answer.classList.remove('selected'));
+    answerElement.classList.add('selected');
```

Comment:
This function is only to add `selected` class name. I think you don't need this function. When user clicks on option you need to add a classname to style the answer if its `wrong` or `correct`. You already have access to selected element inside the click event listener below. Let's discuss this during our meeting